### ISSUE TITLE
Hyperlink message on Commit Status

### ISSUE THREAD

**Issue by mbukosky** [jenkinsci/ghprb-plugin#451]
Is it possible to provide a hyperlink in the message of 'Commit Status Build Result'?

I have tried the following with no luck
* [foo](http://foo.com)
* http://foo.com
* `<a href="foo.com">foo</a>`

I know I can override the 'Details' button but it would be nice to have it in the message itself.

Thanks for the help.

**Comment by benpatterson** [comment]
This might have security constraints by Jenkins core. I'm labeling this as an enhancement though. Thanks for the suggestion. :)

**Comment by mbukosky** [comment]
Yeah, after looking into the GitHub API, I am not sure this can be supported. Feel free to close out if you want.
